hs_model_program/static/js/model_program.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Created by tonycastronova on 10/12/15.
+ */
+
+$alert_error = '<div class="alert alert-danger" id="error-alert"> \
+                    <button type="button" class="close" data-dismiss="alert">x</button> \
+                    <strong>Error </strong> \
+                    failed to retrieve metadata for this resource.\
+                </div>'
```

Comment:
I have changed the variable to `mp_alert_error` so that its value will not be modified outside this js code.  efd0967e01f4135086f657fc430554bd69b59f1c